templates/zerver/integrations.html

```diff
@@ -295,8 +300,8 @@
             <span class="integration-label">Chartbeat</span>
          </a>
        </div>
-       <div class="integration-lozenge integration-circleci">
-         <a class="integration-link integration-circleci" href="https://github.com/hubot-scripts/hubot-circleci" target="_blank">
+       <div class="integration-lozenge integration-circleci-hubot">
+         <a class="integration-link integration-circleci-hubot" href="https://github.com/hubot-scripts/hubot-circleci" target="_blank">
```

Comment:
Yeah, I'd got in mind to ask about removing's possibility.
I don't see any obstacles.